### PR TITLE
HowTo PuTTY, WinSCP and add SSH keys

### PR DESCRIPTION
**Pull request by eleu**
@sbrunner suggested to add a doc page. I began with a first draft here. feedbacks welcome.

### CODE REVIEW

File: doc/integrator/preparativework.rst [hunk 1]
```diff
@@ -0,0 +1,75 @@
+.. _integrator_preparativework:
+
+Creating a SSH Key with Windows
+==================================
+
+1. Download PuTTYgen:
+
+   a) Windows 95, 98, ME, NT, 2000 and XP on Intel x86: http://the.earth.li/~sgtatham/putty/latest/x86/puttygen.exe
```
Comment (asaunier) [inline doc/integrator/preparativework.rst]: What about more recent versions of Windows?
On the download page I don't really see links for 64-bits versions...

File: doc/integrator/preparativework.rst [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+   b) Other operating systems: http://www.chiark.greenend.org.uk/~sgtatham/putty/download.htm
```
Comment (asaunier) [inline doc/integrator/preparativework.rst]: download.htm => download.html

File: doc/integrator/preparativework.rst [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+5. Save the private and public part of the key. Send the public part to the system administrators of the server you want to have access.
+
+Add your SSH Public Key to Github
+==================================
+
+In order to pull and push to Github from your server, you need to add the key to Github: https://help.github.com/articles/generating-ssh-keys/#step-4-add-your-ssh-key-to-your-account
+
+
+Connecting to a server with Windows
```
Comment (asaunier) [inline doc/integrator/preparativework.rst]: with => from
(would be clearer?)

File: doc/integrator/preparativework.rst [hunk 4]
```diff
@@ -0,0 +1,75 @@
+.. _integrator_preparativework:
+
+Creating a SSH Key with Windows
+==================================
```
Comment (asaunier) [inline doc/integrator/preparativework.rst]: Not sure, but shouldn't the === or ---- lines be the width of the title above?

**Comment by asaunier** [comment]
Don't forget to add a reference to your new document in the integrator's index page:
https://raw.githubusercontent.com/camptocamp/c2cgeoportal/master/doc/integrator/index.rst

**Comment by sbrunner** [comment]
Shouldn't be for the version 1.6 ?

**Comment by sbrunner** [comment]
Trailing white spaces: https://travis-ci.org/camptocamp/c2cgeoportal/jobs/72275390#L1353

**Comment by eleu** [comment]
ups something went wrong here... anyway I have to do another pull request.

**Pull request closed without merging**